docs/Embedded/EmbeddedSetup.md

```diff
@@ -0,0 +1,20 @@
+---
+id: EmbeddedSetup
+title: Installing Enklu Embedded
+---
+
+_\[Note: Access to this feature requires a [Studio Subscription](https://enklu.com/pricing). [Contact us](/contact) for details.\]_
+
+![Enklu Embedded](/img/undraw_embedded.svg)
```

Comment:
Awww I love the pink flowers 😍  Makes me think of orchids which very very loosely could relate to our Orchid repo 😂